sdk/servicebus/service-bus/README.md

```diff
@@ -6,13 +6,18 @@ Use the client library `@azure/service-bus` in your application to
 
 - Send messages to an Azure Service Bus Queue or Topic
 - Receive messages from an Azure Service Bus Queue or Subscription
+- Manage Subscription rules
 
-Resources for the v5.0.0-preview.1 of `@azure/service-bus`:
+Resources for the v7.0.0-preview.1 of `@azure/service-bus`:
 
-[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus) | [Package (npm)](https://www.npmjs.com/package/@azure/service-bus) | [API Reference Documentation](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-service-bus/7.0.0-preview.1/index.html) | [Product documentation](https://azure.microsoft.com/en-us/services/service-bus/) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples)
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus) |
+[Package (npm)](https://www.npmjs.com/package/@azure/service-bus) |
+[API Reference Documentation][apiref] |
```

Comment:
When do we update this?

Reply:
Sorry, clarify?

Reply:
When do we update `[API Reference Documentation][apiref]` with an actual link?

Reply:
It is an actual link (it's just at the bottom). NPM and github appear to do the right magic to make it clickable.